docs-src/docs/index.md

```diff
@@ -8,11 +8,12 @@ Layer0 is a framework that helps you deploy web applications to the cloud with m
 Ready to learn more about Layer0? See our [introduction page](intro.md) to learn about some important concepts. When you're ready to get started, take a look at the [installation page](setup/install.md) for information about setting up Layer0.
 
 ## Download
-| Download **v0.10.3** | | 
-| - | - | - |
+| Download **v0.10.3** | | |
```

Comment:
Should this be v0.10.4 or when will this be update?

Reply:
Once we are ready to release, we'll `make release` and push the release onto S3, then this will be updated with that link.

Reply:
I suppose we could do that ahead of time though, since we have control over what the URL path is.